free45/toplevels/bsg_ac_black_parrot_quad_core_rc3/tcl/constraints.tcl

```diff
@@ -153,120 +154,120 @@ if { ${DESIGN_NAME} == "bp_tile_node" || ${DESIGN_NAME} == "bp_processor"} {
-  set ch1_in_dv_port   [remove_from_collection [get_pins -of_objects [get_cells -of_objects [get_ports p_co_*_i]] -filter "name==Y"] $ch1_in_clk_port]
-  set ch1_in_tkn_port                          [get_pins -of_objects [get_cells -of_objects [get_ports p_co_tkn_o]] -filter "name==DATA"]
-  set ch1_out_clk_port                         [get_pins -of_objects [get_cells -of_objects [get_ports p_co2_clk_o]] -filter "name==DATA"]
-  set ch1_out_dv_port  [remove_from_collection [get_pins -of_objects [get_cells -of_objects [get_ports p_co2_*_o]] -filter "name==DATA"] $ch1_out_clk_port]
-  set ch1_out_tkn_port                         [get_pins -of_objects [get_cells -of_objects [get_ports p_co2_tkn_i]] -filter "name==Y"]
-  
-  bsg_comm_link_timing_constraints \
-    ${io_master_clk_name}          \
-    "b"                            \
-    $ch1_in_clk_port               \
-    $ch1_in_dv_port                \
-    $ch1_in_tkn_port               \
-    $ch1_out_clk_port              \
-    $ch1_out_dv_port               \
-    $ch1_out_tkn_port              \
-    100                            \
-    100                           \
-    $io_clk_uncertainty_ns
-
-  # CDC Paths
-  #=================
-  update_timing
-  set clocks [all_clocks]
-  foreach_in_collection launch_clk $clocks {
-    if { [get_attribute $launch_clk is_generated] } {
-      set launch_group [get_generated_clocks -filter "master_clock_name==[get_attribute $launch_clk master_clock_name]"]
-      append_to_collection launch_group [get_attribute $launch_clk master_clock]
-    } else {
-      set launch_group [get_generated_clocks -filter "master_clock_name==[get_attribute $launch_clk name]"]
-      append_to_collection launch_group $launch_clk
-    }
-    foreach_in_collection latch_clk [remove_from_collection $clocks $launch_group] {
-      set launch_period [get_attribute $launch_clk period]
-      set latch_period [get_attribute $latch_clk period]
-      set max_delay_ns [expr min($launch_period,$latch_period)/2]
-      set_max_delay $max_delay_ns -from $launch_clk -to $latch_clk -ignore_clock_latency
-      set_min_delay 0             -from $launch_clk -to $latch_clk -ignore_clock_latency
-    }
-  }
-
-  # Ungrouping
-  #=================
-  set_ungroup [get_cells swizzle]
-
-  set cells_to_derate [list]
-  append_to_collection cells_to_derate [get_cells -quiet -hier -filter "ref_name=~gf14_*"]
-  #append_to_collection cells_to_derate [get_cells -quiet -hier -filter "ref_name=~IN12LP_*"]
-  if { [sizeof $cells_to_derate] > 0 } {
-    foreach_in_collection cell $cells_to_derate {
-      set_timing_derate -cell_delay -early 0.97 $cell
-      set_timing_derate -cell_delay -late  1.03 $cell
-      set_timing_derate -cell_check -early 0.97 $cell
-      set_timing_derate -cell_check -late  1.03 $cell
-    }
-  }
-  #report_timing_derate
-
-  set_dont_touch [get_nets pwrok_lo*]
-  set_dont_touch [get_nets iopwrok_lo*]
-  set_dont_touch [get_nets retc_lo*]
-
+ 
```

Comment:
This is a single clock domain block design right? Can delete all these constraints then